### ISSUE TITLE
Playroom Badge

### ISSUE THREAD

**Issue by hipstersmoothie** [seek-oss/playroom#13]
It would be nice to have a badge to put in the readme to link to the playroom.

Like: https://github.com/storybooks/storybook#badges--presentation-materials

**Comment by markdalgleish** [comment]
Yeah, this is a great idea. I tried making one, and realised that the narrow font I chose for the logo doesn't play well at small sizes. This will need a bit more investigation.

**Comment by markdalgleish** [comment]
If anyone reading this wants to try their hand at designing a new logo and badge combo that doesn't have this issue, please feel free. If you're wondering, the font used for the current logo is [Stint Ultra Expanded](https://fonts.google.com/specimen/Stint+Ultra+Expanded).

**Comment by markdalgleish** [comment]
If you like, you can make a generic badge in the meantime using [shields.io](https://shields.io):

e.g. https://img.shields.io/badge/Try%20it%20with-Playroom-black.svg?style=flat&colorA=707070&longCache=true

**Comment by maraisr** [comment]
Just also wanted to add that if you add the icon to https://github.com/simple-icons/simple-icons then you can simply use `logo=playroom` with shields.io.
